app/models/reservation.rb

```diff
@@ -20,7 +20,7 @@ def date_available?
   end
 
   def outside_date?(date_in_datetime)
-    date_plus_duration_plus_1_day = date_in_datetime + equipment.duration.days + 1.day
+    date_plus_duration_plus_1_day = date_in_datetime + equipment.duration.days
```

Comment:
Can you please change the name of the variable of date_plus_duration_plus_1_day to  date_plus_duration. Since the date_plus_duration is also used on the not_overlapping? method call, probably will be better to add this variable in the date_available? method and pass it over the 2 methods that needed.

Reply:
I understand and made the change. But, about the last thing you said, they're not the same variable since one is for the dates reserved in the array and the other one is for the reserved_date we're trying to compare with

Reply:
Roger!